### ISSUE TITLE
How can I use SASL??

### ISSUE THREAD

**Issue by kimkihoon0515** [jp-gouin/helm-openldap#144]
@jp-gouin  I have a simple question.
Refer to https://www.openldap.org/doc/admin26/guide.html
```
14.5. Pass-Through authentication
Since OpenLDAP 2.0 slapd has had the ability to delegate password verification to a separate process. This uses the sasl_checkpass(3) function so it can use any back-end server that Cyrus SASL supports for checking passwords. The choice is very wide, as one option is to use saslauthd(8) which in turn can use local files, Kerberos, an IMAP server, another LDAP server, or anything supported by the PAM mechanism.

The server must be built with the --enable-spasswd configuration option to enable pass-through authentication.
```
However i can't find how to activate this option before starting openldap server
Or is it already done?
